frontend/www/js/omegaup/components/group/Identitiesv2.vue

```diff
@@ -103,11 +107,7 @@ export default class Identities extends Vue {
       ui.error(T.groupsInvalidCsv);
       return;
     }
-    this.$emit('read-csv', this, fileUpload);
-  }
-
-  onBulkIdentities(): void {
-    this.$emit('bulk-identities', this, this.identities);
+    this.$emit('read-csv', this.data, fileUpload);
```

Comment:
```suggestion
    this.$emit('read-csv', { identities: this.identities }, fileUpload);
```

es mejor poner el objeto acá para evitar tener propiedades mutables que deberían ser cosas simples independientes en objetos complejos. la ventaja de tenerlos separados en el componente es que así se puede tener watchers independientes para cada propiedad. a pesar de que no hay `@Watch` explícitos, Vue tiene que andar viendo que cada vez que se usa `data.xxx` y se actualiza _cualquier_ propiedad de `data`, tiene que recalcular todo lo que `data` toque.

Reply:
Esto va a ayudar mucho para dejar de mandar todo el componente por todos lados. Gracias.